src/types/auth.d.ts

```diff
@@ -1,34 +1,35 @@
 import { User } from './backend'
 
-export type CredentialsLogin = {
+export interface CredentialsLogin {
   email: string
   password: string
+  username?: string
 }
 
 export type UserCredentialsProp = Pick<
```

Comment:
I'm not sure what's this actually? I don't see credentials...

Reply:
It's definitely named strange because `first_name` or `last_name` aren't really credentials. 😄